src/serverMain/kotlin/com/studo/campusqr/database/InitialSetup.kt

```diff
@@ -2,15 +2,23 @@ package com.studo.campusqr.database
 
 import com.studo.campusqr.common.UserType
 import com.studo.campusqr.extensions.runOnDb
+import com.studo.campusqr.serverScope
 import com.studo.campusqr.utils.Algorithm
+import com.studo.katerbase.equal
 import com.studo.katerbase.sha256
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
 import java.util.*
 import kotlin.streams.toList
 
+const val DEMO_MODE = 1
```

Comment:
I don't understand this variable. 1. why is this an integer? 2. how is this different to `val demoMode`? 3. Why is this not database-changeable?